docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
@nwt That helps, thanks. Not sure if it's the wording itself or just the fact I understood that you were clarifying between two distinct scenarios but 👍.